Pod/Classes/ReactiveCocoa/SignalProducer+Argo.swift

```diff
@@ -20,13 +20,13 @@ public extension SignalProducerProtocol where Value == Moya.Response, Error == M
      
      - returns: returns Observable of mapped objects
      */
-    public func mapObject<T:Decodable where T == T.DecodedType>(type: T.Type, rootKey: String? = nil) -> SignalProducer<T, Error> {
+    public func mapObject<T:Decodable>(type: T.Type, rootKey: String? = nil) -> SignalProducer<T, Error> where T == T.DecodedType {
         
         return producer.flatMap(.latest) { response -> SignalProducer<T, Error> in
             
             do {
                 return SignalProducer(value: try response.mapObject(rootKey: rootKey))
-            } catch let error as Moya.Error {
+            } catch let error as Moya.MoyaError {
```

Comment:
Can probably drop the `Moya.` prefix now, since it was just clarifying between `Moya.Error` and `Swift.Error`